### ISSUE TITLE
Simplelogin alias generation only generate random words instead the domain name

### ISSUE THREAD

**Issue by Fred-Vaniss** [bitwarden/clients#12461]
### Steps To Reproduce

0. Have a Simplelogin account and an API key
1. Open a browser tab on a website
2. Create a new login on the browser extension
3. Click on "generate username"
4. Select "Forwared email alias" then "Simplelogin" service
5. Enter the API code
6. Generate the new alias

### Expected Result

The alias generated with `website.abc12@simplelogin.com` based on the current browser tab.

It should have generated an alias based on the current website domain name and fill the Simplelogin's alias note `Website: www.website.com. Generated by Bitwarden`

### Actual Result

The alias only generate `random_word123@simeplogin.com`

It only generate two random words and three numbers and no longer specify on what website the alias was generated on Simplelogin's alias note

### Additional Context

Tested on both the new version on Chrome and the old version on Firefox. The old version on Firefox still works as expected.

### Operating System

Windows

### Web Browser

Chrome

### Browser Version

Chrome for developers 133.0.6888.2

### Build Version

2024.12.2

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by daniellbw** [comment]
Hi there,

I am unable to reproduce this issue, it has been escalated for further investigation. If you have more information that can help us, please add it below. 

Thanks!

**Comment by Fred-Vaniss** [comment]
> Hi there,
> 
> I am unable to reproduce this issue, it has been escalated for further investigation. If you have more information that can help us, please add it below.
> 
> Thanks!

I don't have more information, just the fact this issue appeared on the new UI re-design 2024.12 (only available on Chromium browsers at the moment). Also I tested it on Chrome for Developers edition.

**Comment by antoniorecano** [comment]
Also happens to me on safari.

**Comment by martijnkriebel** [comment]
Same problem after the update of the extension on my Chrome browser (version 131.0.6778.205). BW/SimpleLogin generates an email address with random string of characters and numbers separated by dashes, instead of using the domain name.

**Comment by Fred-Vaniss** [comment]
Just tested on Firefox 133.0.3, same issue.

**Comment by dmezrahy** [comment]
Been having the same issue on Firefox 133.0.3 since the latest Bitwarden browser extension update (just tested on version 2024.12.3, issue persists).

**Comment by radutomy** [comment]
Same issue here. Using Firefox.

**Comment by rfc2119** [comment]
Can reproduce on Firefox 133.0.3 (x86 64-bit)

**Comment by rpc31** [comment]
same issues here. Sometimes even, when trying to create a simplelogin alias, it crashes the extension. Can only works if kill then reboot safari.

**Comment by the-sphynx** [comment]
Just stating that I see it in the Android app as well, that it only generates the random words and doesn't include the domain for the site you create.  Hoping it's an easy fix that gets closed out soon.

**Comment by SimplyBacon** [comment]
I also have this issue on the Chrome Web extension using Brave and on the Android app.  Seeing as how this has been an issue for two months can we escilate the priority?

**Comment by audreyality** [comment]
> I have some tech debt PRs landing, after which restoring website support is my next priority.

We were able to boost the priority of a fix! This only restores the website behavior within the forwarders. Website support for catchall and plus-addressed emails is still pending.

**Comment by SimplyBacon** [comment]
@audreyality Thank you, will this address the Android app as well?

**Comment by audreyality** [comment]
No. They have an independent forwarder implementation. One day we'll share it, but there's no timeline for that work yet.

**Comment by radutomy** [comment]
This is not fixed for me. When I generate an username using a simplelogin API key, I'm expecting to see something like `github@mydomain.com` instead I get `random-word@mydomain.com`

**Comment by s3igo** [comment]
@radutomy I believe this fix is included in version v2025.2.2 or later. However, in my browser environment, the latest version available on the Firefox Add-ons store is still v2025.2.0 (while the Chrome Web Store has already been updated to v2025.2.2).
If you're seeing the same situation, I think you'll need to download and install it directly from the GitHub Releases page.

**Comment by radutomy** [comment]
@s3igo Thanks for confirming. I'll patiently await for it to make its way to the Firefox store.

**Comment by TechKnowCase** [comment]
The Firefox addon hasn't been updated in the last 2 months. I do see releases in the project releases tab. Is this normal for the firefox store to be this slow?
